pep-9999.rst

```diff
@@ -0,0 +1,670 @@
+
+
+License Expression Library Reference implementation
+===================================================
+
+To be succesful, `License Expression` need to be supported in tools.
+
+The `license-expression library`_ provides a Python reference implementation
+library to handle `License Expressions` including parsing, validating and
+formatting `License Expressions` using flexible list of license symbols
+(including SPDX license identifiers and extra identifiers referenced here).
+It is licensed under the Apache-2.0 license and is used in a few projects such
+as the SPDX Python tools, the ScanCode toolkit and the FSFE Reuse.software
+project.
+
+If this PEP is accepted, the Python packaging authority should consider
+adopting the `license-expression library`_ as one of its own tools
+<https://github.com/pypa>_.
+
+
+Surveying how we document licenses today in Python
+==================================================
+
+There are multiple ways used or recommended to document Python package
+licenses today:
+
+
+In Core metadata
+----------------
+
+There are two overlapping Core metadata fields to document a license: the
+license-related `Classifiers <https://pypi.org/classifiers/>_` prefixed with
+`License::` and the `License
+<https://packaging.python.org/guides/distributing-packages-using-setuptools/?highlight=MANIFEST.in#license>_`
+field as free text.
+
+The Core metadata documentation `License` field documentation is currently::
+
+    License (optional)
+    ::::::::::::::::::
+
+    Text indicating the license covering the distribution where the license
+    is not a selection from the "License" Trove classifiers. See
+    "Classifier" below.  This field may also be used to specify a
+    particular version of a license which is named via the ``Classifier``
+    field, or to indicate a variation or exception to such a license.
+
+    Examples::
+
+        License: This software may only be obtained by sending the
+                author a postcard, and then the user promises not
+                to redistribute it.
+
+        License: GPL version 3, excluding DRM provisions
+
+Even though there are two fields, it is at times difficult to convey anything
+but simpler licensing. For instance some `Classifiers` lack accuracy (GPL
+without a version) and when you have multiple License-related classifiers it is
+not clear if this is a choice or all these apply and which ones. Furthermore,
+the list of available license-related `Classifiers` is often out-of-date.
+
+
+In the pypa sample project
+--------------------------
+
+The latest pypa sample project recommends only to use `Classifiers in setup.py
+<https://github.com/pypa/sampleproject/blob/b0d3f3eeef4e5668d7b59448b43c0f1914d9afc6/setup.py#L103>_`
+and does not list the `license` field in its example `setup.py`.
+
+
+The License files in wheels and setuptools
+------------------------------------------
+
+Betyond a license code or qualifier, license text files are documented and
```

Comment:
```suggestion
Beyond a license code or qualifier, license text files are documented and
```

Reply:
Fixed! thank you for this and the other typo fixes.